srcjs/input_binding_radio.js

```diff
@@ -5,10 +5,25 @@ $.extend(radioInputBinding, {
   },
   getValue: function(el) {
     // Select the radio objects that have name equal to the grouping div's id
-    return $('input:radio[name="' + $escape(el.id) + '"]:checked').val();
+    var value = $('input:radio[name="' + $escape(el.id) + '"]:checked').val();
+    // If none are checked, the input will return null (it's the default on load,
+    // but it wasn't emptied when calling updateRadioButtons with character(0)
+    if (value) {
+      return value;
+    } else {
+      return null;
+    }
   },
   setValue: function(el, value) {
-    $('input:radio[name="' + $escape(el.id) + '"][value="' + $escape(value) + '"]').prop('checked', true);
+    if (value === undefined) {
+      return;
+    } else if (value.length === 0) {
+      // Removing all checked item if the sent data is empty
+      $('input:radio[name="' + $escape(el.id) + '"]').prop('checked', false);
+    } else {
+      $('input:radio[name="' + $escape(el.id) + '"][value="' + $escape(value) + '"]').prop('checked', true);
+    }
```

Comment:
Note that an ideal test app should touch all 3 cases here

Reply:
@cpsievert I'm not sure when `value` could be undefined here?

Reply:
If we don't know when it could be `undefined`, then let's take out the check. If some weird data gets sent to this function, better if it throws an error and we find out about it.